manager/controllers/paging.go

```diff
@@ -0,0 +1,66 @@
+package controllers
+
+import "fmt"
+
+func CreateLinks(path string, offset, limit, total int, dataFormat string, showAll bool) Links {
```

Comment:
What's the reason for show_all parameter ?

Reply:
to show all systems or only for current account

Reply:
It should allow you to show all Advisories, not only those that are affecting your environment. Currently it's not a requirement so we can remove it.